web/styles/compose.css

```diff
@@ -458,7 +458,7 @@
     margin: 0 1px;
 }
 
-#compose-limit-indicator:not(:empty) {
+.message-limit-indicator:not(:empty) {
```

Comment:
The refactor to add this class for the existing compose box should be its own little prep commit that's atomically correct. That structure really helps reviewers, because you're promising that the first tiny commit (or maybe a few refactor commits like it) is/are the only one that would change compose box behavior, and then the much larger commit(s) adding the edit feature would be designed to be obviously only impacting the message-edit UI.